File: c++/Easy/Part003/Convert Rupee to Paise.cpp
#include <iostream>
#include <cmath> // for round

int main() {
    double rupees;
    std::cin >> rupees;
    
    // Convert rupees to paise and round the result to the nearest integer
    int paise = static_cast<int>(std::round(rupees * 100));
    
    // Output the paise value
    std::cout << paise << std::endl;
    
    return 0;
}

File: Java/Easy/Part003/Convert Rupee to Paise.java
import java.util.Scanner;

public class Hello {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double rupees = scanner.nextDouble();

        // Convert rupees to paise and round the result to the nearest integer
        int paise = (int) Math.round(rupees * 100);

        // Output the paise value
        System.out.println(paise);

        scanner.close();
    }
}
